src/transformers/models/marian/configuration_marian.py

```diff
@@ -97,3 +97,4 @@ class MarianConfig(BartConfig):
     """
 
     model_type = "marian"
+    keys_to_ignore_at_inference = ["past_key_values"]
```

Comment:
It's a bit late now, but I'm not a huge fan of the name to be honest -> this seems to be very specific to training, but one might think now that `past_key_values` can never be passed during inference in general. Why not call it `keys_to_ignore_at_training`?

Reply:
No this is not for training, only for inference. During training we only get the loss in the outputs.
And this is not ignore to pass to the model, but ignore because they are not part of the logits/scores/predictions we want to gather. Maybe `output_keys_to_ignore_at_inference` is clearer?

Reply:
I see! Yeah I think `output_keys_to_ignore_at_inference` would be a bit clearer to me :-)

Reply:
See #8857